exodus/trackers/tasks.py

```diff
@@ -0,0 +1,57 @@
+import logging
+
+from celery import shared_task
+from django.conf import settings
+from eventlog.events import EventGroup
+
+from trackers.models import Tracker
+
+
+logger = logging.getLogger(__name__)
+
+
+@shared_task
+def auto_update_trackers():
+    import urllib.request
+    import json
```

Comment:
Can we use `requests` instead of `urllib.request`? The code will be way more readable
I think we may not even need `json` in this case

Reply:
I'll check that.